javascript-functions-practice/get-last-element.js

```diff
@@ -0,0 +1,5 @@
+/* exported getLastElement */
+function getLastElement(array) {
+  var lastElement = array.pop();
```

Comment:
This will remove the last item of the array permanently, which is bad news 💀. You should instead retrieve the value without changing the original array